### PR TITLE
Monitord thermal errors redmine7238

### PR DESCRIPTION
**Pull request by jimis**

### CODE REVIEW

File: cf-monitord/mon_temp.c [hunk 1]
```diff
@@ -86,10 +86,17 @@ void MonTempInit(void)
 {
     struct stat statbuf;
 
-    if (stat("/sys/devices/virtual/thermal", &statbuf) != -1)
+    for (int i = 0; i < 4; i++)
```
Comment (kacfengine) [inline cf-monitord/mon_temp.c]: Why 4?

Comment (kacfengine) [inline cf-monitord/mon_temp.c]: Oh, I see, it only supports four of them.

Comment (jimis) [inline cf-monitord/mon_temp.c]: yes, it's already hardcoded elsewhere. It was to much for me to change at this point. :-)

**Comment by kacfengine** [comment]
Looks good. I suppose none of this stuff has an acceptance test anyway, so if you tested it locally, just merge it.

**Comment by jimis** [comment]
@kacfengine great thanks for reviewing!

**Pull request merged**